frontend/src/components/card/ProblemCard.tsx

```diff
@@ -43,6 +43,14 @@ const TitleText = styled(LargeText)`
   vertical-align: middle;
 `;
 
+const DescriptionText = styled(Text)`
+  display: -webkit-box;
+  -webkit-line-clamp: 1;
+  overflow: hidden;
+  -webkit-box-orient: vertical;
+  word-break: break-all;
+`;
+
```

Comment:
nit: maybe rename and move this to the Text file so it can be reused? Could potentially have the font size as inherit so that it automatically adjusts to whatever wraps it - problem with this is that I don't think you're allowed/supposed to nest p tags (maybe changing this to span will help with that, but idk, if that doesn't work this should be fine as is)

Reply:
I added a `SingleLineText` component to the `Text.tsx` file for this - I saw you already implemented your own implementation in #174, so whether you use this or not is fine by me. 👍